src/installer/tests/HostActivation.Tests/MultilevelSDKLookup.cs

```diff
@@ -559,19 +556,17 @@ private void AddAvailableSdkVersions(string sdkBaseDir, params string[] availabl
                 File.WriteAllText(Path.Combine(newSdkDir, "dotnet.dll"), string.Empty);
 
                 // ./dotnet.runtimeconfig.json
-                string runtimeConfig = Path.Combine(newSdkDir, "dotnet.runtimeconfig.json");
-                File.Copy(dummyRuntimeConfig, runtimeConfig, true);
+                RuntimeConfig.FromFile(Path.Combine(newSdkDir, "dotnet.runtimeconfig.json"))
+                    .WithFramework(Constants.MicrosoftNETCoreApp, "9999.0.0")
+                    .Save();
             }
         }
 
         // Put a global.json file in the cwd in order to specify a CLI
-        private string SetGlobalJsonVersion(string globalJsonFileName)
+        private string SetGlobalJsonVersion(string version)
```

Comment:
Nit: the method is now identical to the one in the other test, so maybe we could dedupe it somehow - but it's not important.